pkg/controller/capacity/capacity_controller.go

```diff
@@ -41,64 +39,83 @@ const (
 	AgentName   = "capacity-controller"
 	SadPodLimit = 5
 
-	ClustersNotReady = "ClustersNotReady"
-	InProgress       = "InProgress"
-	InternalError    = "InternalError"
-	PodsNotReady     = "PodsNotReady"
-	DeploymentStuck  = "DeploymentStuck"
+	InProgress      = "InProgress"
+	InternalError   = "InternalError"
+	PodsNotReady    = "PodsNotReady"
+	DeploymentStuck = "DeploymentStuck"
 
-	CapacityTargetConditionChanged  = "CapacityTargetConditionChanged"
-	ClusterCapacityConditionChanged = "ClusterCapacityConditionChanged"
+	CapacityTargetConditionChanged = "CapacityTargetConditionChanged"
 )
 
 // Controller is the controller implementation for CapacityTarget resources
 type Controller struct {
-	clientset shipperclient.Interface
-	store     clusterclientstore.Interface
+	shipperClient shipperclient.Interface
+	kubeClient    kubernetes.Interface
 
 	capacityTargetsLister listers.CapacityTargetLister
 	capacityTargetsSynced cache.InformerSynced
 
-	releasesLister       listers.ReleaseLister
-	releasesListerSynced cache.InformerSynced
+	deploymentsLister appslisters.DeploymentLister
+	deploymentsSynced cache.InformerSynced
```

Comment:
Do we care about `ReplicaSets` here?

Reply:
No, we only directly interact with deployments. Any interesting changes to the ReplicaSet would reflect on the Deployment anyway, so just watching them is sufficient.